### PR TITLE
Overture places

### PR DESCRIPTION
**Pull request by Bonkles**
Okay, this is the beginnings of an overture 'Service' that can reach out to overture s3 buckets, read a manifest, and display overture places data. 

https://github.com/user-attachments/assets/7d6bfdab-5ce6-4ea3-b70b-82d72a8e7013

### CODE REVIEW

File: modules/pixi/PixiLayerOverture.js
Comment (bhousel) [inline modules/pixi/PixiLayerOverture.js:1]: I'm ok with Overture to be its own layer, but I wonder why not just use the PixiLayerRapid.js layer to display them?  Would users need to toggle the "Overture Stuff" and "Rapid Stuff" layers independently of each other?   

It seems more like the Overture sources could just be considered additional Rapid datasets (like the Esri sources), and you do have them set up this way in the RapidSystem.

File: modules/services/OvertureService.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  /**
+   * initAsync
+   * Called after all core objects have been constructed.
+   * @return {Promise} Promise resolved when this component has completed initialization
+   */
+  initAsync() {
+    const context = this.context;
+    const vtService = context.services.vectortile;
+
+    this._loadS3Catalog();
```
Comment (bhousel) [inline modules/services/OvertureService.js:59]: How you have it written, the catalog fetch isn't tied to this init promise really.
It should maybe be something like:
```js
return Promise.all([
  vtService.initAsync(),
  this._loadS3Catalog()
]); 
```  

Then this service isn't considered "initted" until both things happen.

File: modules/services/OvertureService.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+      dateStrings.sort( (a, b) => new Date(b) - new Date(a));
+
+      // Grab the very latest date stamp and keep track of the release associated with it.
+      this.latestRelease = this.pmTilesCatalog.releases.find(release => release.release_id === dateStrings[0]);
+
+    });
+  }
+
+  loadStacMetadata () {
```
Comment (bhousel) [inline modules/services/OvertureService.js:91]: I think you were saying this goes away, right?

**Review by bhousel — APPROVED** [review]
Seems like a good start!  Looking forward to adding some more features to make this data easy for users to inspect and work with 👍

File: modules/pixi/PixiLayerOverture.js
Comment (Bonkles) [reply modules/pixi/PixiLayerOverture.js:1]: Eventually, I see us making a completely separate rendering ruleset for Overture data (i.e. a completely new styles.json specifically for overture, overture specific entity rendering stuff, etc) that might start to overload the Rapid renderer. 

For the near term, though, if all we're going to render is places, I'm okay with just putting this stuff inside the PixiLayerRapid folder with the idea that we extract it out when the time comes.

File: modules/services/OvertureService.js (hunk 2 above)
Comment (Bonkles) [reply modules/services/OvertureService.js:91]: yeah this method needs a rename- it should just be something like 'loadJsonManifest'.

Comment (Bonkles) [reply modules/services/OvertureService.js]: Update: I stand corrected, this method just straight up should be deleted.

Comment (Bonkles) [reply modules/services/OvertureService.js]: Done!

File: modules/pixi/PixiLayerOverture.js
Comment (bhousel) [reply modules/pixi/PixiLayerOverture.js:1]: I'm not wild about having 2 layers that do the same thing (where one just does `if overture` and the other one `if not overture`) , but we can merge as-is and see how it develops.

**Pull request merged**